ocaml/typing/typeclass.ml

```diff
@@ -1357,6 +1359,7 @@ and class_expr_aux cl_num val_env met_env virt self_scope scl =
       in
       let (args, cty) =
         let (_, ty_fun0) = Ctype.instance_class [] cl.cl_type in
+        let sargs = List.map (fun (label, e) -> transl_label label, e) sargs in
```

Comment:
Not sure offhand how to trigger this one. But I'm sure it suffers the same problems as the others.